lint-checks-android/src/test/java/com/uber/lintchecks/android/XmlWebViewInsideScrollViewDetectorTest.kt

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright (C) 2019. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.lintchecks.android
+
+import com.android.tools.lint.checks.infrastructure.TestLintTask
+import com.uber.lintchecks.base.test.LintTestBase
+import org.junit.Test
+
+class XmlWebViewInsideScrollViewDetectorTest : LintTestBase() {
+  @Test
+  fun testDetector_scrollViewWithoutFillViewPort_shouldFail() {
+    TestLintTask.lint()
+        .files(xmlSource("res/layout/webview_inside_scrollview_wo_fill_viewport.xml",
+            """<?xml version="1.0" encoding="utf-8"?>
+                            <ScrollView
```

Comment:
Nit: These are indented weird. <ScrollView> and <?xml version> should be on the same indent level

Reply:
I see, let me format them

Reply:
@ShaishavGandhi i ran a reformat on the file. Still this indent remained

Reply:
Done. Manually formatted